libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -259,6 +345,17 @@ - (void)setAccountPersister:(id<SFUserAccountPersister>) persister {
     }
 }
 
+- (SFOAuthCredentials *)retrieveClientCredentials{
+    NSString *identifier = self.oauthClientId;
+    SFOAuthCredentials *creds = [[SFOAuthCredentials alloc] initWithIdentifier:identifier clientId:self.oauthClientId encrypted:YES];
```

Comment:
The identifier passed to `SFOAuthCredentials` is supposed to be unique across the app instance.  It's not going to be here, as defined.